aggregator/src/aggregator/http_handlers.rs

```diff
@@ -141,8 +142,19 @@ async fn run_error_handler(error: &Error, mut conn: Conn) -> Conn {
                 .with_task_id(&inner.task_id)
                 .with_detail(&inner.to_string()),
         ),
+        Error::Datastore(inner) => match inner {
+            datastoreError::TimeUnaligned(error) => conn.with_problem_document(
```

Comment:
as observed elsewhere, try:
```suggestion
        Error::Datastore(datastoreError::TimeUnaligned(error)) => conn.with_problem_document(
```
...and put the `Error::Datastore(_)` case back, below. I think it's prettier, but it's subjective.
(this suggestion won't lint or even compile unfortunately)

Reply:
Matching syntax has so, so much to it. Very cool.